### PR TITLE
ABC-53: About Page

### PR DESCRIPTION
**Pull request by KevinLaCarrubba**
- Updated firebase about-page collection to contain 2 docs
  - The doc fields are as follows: 
  - `{info, header, subHeader, title, role, name, profileImg}`
- Created a client facing page which maps over the about page data and returns 2 sections in which all the info is stored.

*****
**To Do**
- Mobile Responsive Styling
*****
We can re visit the field names for the db

### CODE REVIEW

File: lib/db/cms/about-page.ts [hunk 1]
```diff
@@ -5,22 +5,24 @@ import { AboutPageProps } from '@/utils/types/redux'
...
-            role
-        }
 
-        return aboutPageData
+        return data.docs.map(doc => {
+            const { info, header, profileImg, role, subHeader, title, name } =
+                doc.data()
+            return {
+                id: doc.id,
+                info,
```
Comment (arfusop) [inline lib/db/cms/about-page.ts:17]: My only qualm here is `info` should really be `description`

File: pages/contact.tsx [hunk 2]
```diff
@@ -1,3 +1,3 @@
 export default function ContactPage() {
-    return <div>Contact Page placeholder</div>
+    return <div>Contact Page holder</div>
```
Comment (arfusop) [inline pages/contact.tsx:2]: I cant believe you even wasted energy on this lmao

**Review by arfusop — CHANGES_REQUESTED** [review]
We can delete the `assets/about-me.svg` image. One other small thing, but looking excellent

File: pages/contact.tsx (hunk 2 above)
Comment (KevinLaCarrubba) [reply pages/contact.tsx:2]: I put the whole about me file in there 🤣 and commited it then im clicking the link going nuts like wtf and it was in the wrong file. so i had to re do it 🤣😂

Comment (arfusop) [reply pages/contact.tsx:2]: LOL

**Review by arfusop — APPROVED** [review]

**Pull request merged**